zulipterminal/ui_tools/boxes.py

```diff
@@ -107,15 +107,20 @@ def generic_autocomplete(self, text: str, state: int) -> Optional[str]:
             if text.startswith(prefix):
                 typeaheads = autocomplete_func(text, prefix)
                 reduced_typeaheads = typeaheads[:num_suggestions]
+                truncated = len(reduced_typeaheads) != len(typeaheads)
```

Comment:
It'd be useful to prefix this and similar boolean values with a `is_` (or sometimes `in_` or similar)